cypress/e2e/newContactSpec.cy.js

```diff
@@ -373,5 +373,22 @@ describe("New Contacts page after logging in", () => {
       );
       cy.contains("button", "X").scrollIntoView().click();
     });
+
+    it("Should keep the add company modal open when clicked", () => {
+      cy.get("a > .bg-cyan-600").click();
+      cy.get("#firstName").type("Emma");
+      cy.get("#lastName").type("Boots");
+      cy.contains("Add New Contact");
+      cy.contains("button", "Add new company").click();
+      cy.wait(5000)
```

Comment:
I would add a comment here about why the `wait` exists. It's not something you often see in tests, and I could see it being removed by someone in the future who didn't understand that there is actually a reason for it.